test/unit_tests/sizingFractionsTest.cpp

```diff
@@ -22,7 +22,7 @@ TEST(SizingFractionsTest, TamScalable_SP_SizingFract)
     double aquaFrac, useableFrac;
     const double aquaFrac_answer = 4. / logicSize;
 
-    EXPECT_EQ(hpwh.getSizingFractions(aquaFrac, useableFrac), 0);
+    EXPECT_NO_THROW(hpwh.getSizingFractions(aquaFrac, useableFrac));
```

Comment:
`EXPECT_NO_THROW` isn't necessary since an exception would also cause the test to fail.